src/services/UserService.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import humps from 'humps';
+import api from '../config/api';
+import type { User } from '../models/User';
+
+export function authHeader() {
+  // return authorization header with jwt token
+  const user = JSON.parse(localStorage.getItem('user'));
```

Comment:
Hmmm would be good to use the store here too...
But we cannot really access it here.
Maybe it's okay for now but we should really consider using some kind of middleware to update our axios instance.

Reply:
I agree on the middleware, but I think we should postpone that for when we actually want to protext routes. Until then I will store the user in local storage for simplicity